renderdoc/os/os_specific.h

```diff
@@ -248,6 +248,8 @@ bool GetLoadedModules(char *&buf, size_t &size);
 
 namespace FileIO
 {
+void GetPackageName(string &selfName);
```

Comment:
I don't think a new function is needed here with an android-specific concept. If android doesn't have the same concept of executables I think it would be fine to return the package name under the existing GetExecutableFilename instead of whatever it currently does, that way other code that wants to get some kind of 'current executing program' like the default capture filename will also work.